pyanaconda/ui/lib/help.py

```diff
@@ -262,4 +263,12 @@ def show_graphical_help(help_path, help_anchor=None):
     else:
         args.append(help_path)
 
-    yelp_process = startProgram(["yelp", *args], reset_lang=False)
+    # Let's see if we can run yelp under liveuser on live
+    # See: https://bugzilla.redhat.com/show_bug.cgi?id=2118832
+    try:
+        user = getpwnam("liveuser")
```

Comment:
I would prefer to limit this functionality to live environments, just in case. We would need a new rule for that, so we can check something like `conf.system.provides_live_user`.